infrastructure/ArmTemplates/antareskvtemplate.json

```diff
@@ -334,7 +334,7 @@
               },
               {
                 "Name": "ASPNETCORE_ENVIRONMENT",
-                "Value": "Staging"
+                "Value": "Production"
```

Comment:
How will this work when we deploy to staging slot? wont the value be set to Production there?

Reply:
https://github.com/Azure/Azure-AppServices-Diagnostics/blob/5b3edd6eceedcc0dbeb7530a7a18e942c44d7258/infrastructure/ArmTemplates/antareskvtemplate.json#L390

There is a different variable for staging slot